internal/wrappers/feature-flags.go

```diff
@@ -55,26 +58,70 @@ var FeatureFlagsBaseMap = []CommandFlags{
 }
 
 var FeatureFlags = map[string]bool{}
+var FeatureFlagsSpecific = map[string]bool{}
 
 func HandleFeatureFlags(featureFlagsWrapper FeatureFlagsWrapper) error {
 	allFlags, err := featureFlagsWrapper.GetAll()
 	if err != nil {
-		loadFeatureFlagsDefaultValues()
-
+		LoadFeatureFlagsDefaultValues()
 		return nil
 	}
 
 	loadFeatureFlagsMap(*allFlags)
-
 	return nil
 }
 
+func GetSpecificFeatureFlag(featureFlagsWrapper FeatureFlagsWrapper, flagName string) (*FeatureFlagResponseModel, error) {
+	if value, exists := FeatureFlagsSpecific[flagName]; exists {
+		return &FeatureFlagResponseModel{Name: flagName, Status: value}, nil
+	}
+
+	specificFlag, err := getSpecificFlagWithRetry(featureFlagsWrapper, flagName, maxRetries)
+	if err != nil {
+		UpdateSpecificFeatureFlagMapWithDefault(flagName)
```

Comment:
I think we dont need UpdateSpecificFeatureFlagMapWithDefault, we can just return FeatureFlags[flagName] (because it's error)

Reply:
We need to update the cache because the new logic in GetSpecificFeatureFlag
ensures that if an item is found in the map (cache), no HTTP request is made.

This way, we avoid making multiple HTTP calls.

Reply:
UpdateSpecificFeatureFlagMapWithDefault is delete